source/_docs/automation/editor.markdown

```diff
@@ -72,8 +72,8 @@ You can use the `automation:` and `automation old:` sections at the same time:
  - `automation:` to save the automation created by the online editor
 
 ```yaml
-automation: !include automations.yaml
-automation old: !include_dir_merge_list automations
+automation: !include_dir_merge_list automations/
```

Comment:
Please remove this change. It doesn't belong to the Xiaomi Vacuum feature, right?

Reply:
Sorry. I opened a issue and pushed it incorrectly